ts/src/kraken.ts

```diff
@@ -1540,15 +1547,6 @@ export default class kraken extends Exchange {
         //
         const result = response['result'];
         const trades = result[id];
-        // trades is a sorted array: last (most recent trade) goes last
-        const length = trades.length;
-        if (length <= 0) {
-            return [];
-        }
-        const lastTrade = trades[length - 1];
-        const lastTradeId = this.safeString (result, 'last');
-        lastTrade.push (lastTradeId);
-        trades[length - 1] = lastTrade;
```

Comment:
Please don't remove this.

this is critical for PROPER pagination in accordance with [kraken's docs](https://docs.kraken.com/api/docs/rest-api/get-recent-trades).